packages/create-cloudflare/src/helpers/shell-quote.ts

```diff
@@ -1,6 +1,10 @@
 import shellquote from "shell-quote";
 
-export const quote = shellquote.quote;
+export const quote = function (args: (string | number | boolean)[]) {
```

Comment:
is this needed? doesn't `shellquote.quote` perform the mapping to `String()` for you?
(https://github.com/ljharb/shell-quote/blob/fdeb0fd102ecdd4ff227a978532598c462312347/quote.js#L14)

Reply:
I see.... well if it's just for types sake maybe I would just "fix" the quite signature instead, something like:
```
export const quote = shellquote.quote as (args: unknown[]) => string;
```
(maybe + a comment saying that the shell-quote type signature is incorrect)
and let shell-quote do its thing

anyways it's no big deal either version looks more or less equivalent to me 🙂:+1:

Reply:
> in case they consider their types as the public API and break the implementation later
ah no yeah... good point 😕👍